plugins/MantisGraph/MantisGraph.php

```diff
@@ -113,6 +113,7 @@ function resources() {
 			html_javascript_link( 'chart-' . CHARTJS_VERSION . '.min.js' );
 			html_javascript_link( 'chart.bundle-' . CHARTJS_VERSION . '.min.js' );
 		}
+		html_javascript_link( '../plugins/MantisGraph/MantisGraph.js' );
```

Comment:
Unless they are actually used somewhere else, I would rather move all js files to the plugin's file directory.

We can revisit this if we realize later on that some of these scripts are used elsewhere.

One could also argue that a dashboard should be a part of the Graph plugin itself.